cEP-0000.md

```diff
@@ -4,7 +4,7 @@ coala Enhancement Proposals
 |Metadata|                                   |
 |--------|-----------------------------------|
 |cEP     |0000                               |
-|Version |2.1                                |
+|Version |2.2                                |
```

Comment:
@Asnelchristian why? The version change belongs to the rest IMO. I don't wanna make this harder for @arnab1896 because it doesn't matter too much so either is fine with me now.